### ISSUE TITLE
Add an advanced search Boolean option to identify Works identified as Public Domain (Yes/No) (from 1850)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#47]
**Problem Description**: A feedback request came in to add a facet for public domain materials. YUL does not currently provide this designation, which is why LUX does not offer that facet.  The Rights WG discussed on 6/15/23 and would like to revisit issue as it would be very beneficial for museums.  Discussed at Rights on 9/26 - agreement to make structure rights data available to Adv. Search, but not expose a new facet.   Subsequent conversations illuminated the complexity around this.

**Expected behaviors**: 
Users Should be able to use Advanced Search to pinpoint objects that have been explicitly stated as being in the public domain.  Users should see help text that clarifies that this public domain attribution ONLY applies to materials from YUAG and YCBA at this point.  There should be a link to the Rights and Usage FAQ from the Advanced Search help. 

**Requirements:**
- Update Subject To Label to "Rights Status"
- Add a new Advanced Search option on Works "Are in the Public Domain" (Yes/No checkbox)
- Display new help text (below)
- Help text should include a link to the Rights and Usage FAQ

**Help Text** (confirmed by Rights WG)
_Search for Works that have been identified as public domain/no copyright materials.  Note that this only applies to materials from the Yale University Art Gallery and Yale Center for British Art collections at this time._

_For copyright questions or more information about rights and re-use of content found in LUX, please contact the appropriate museum or library or consult the [Rights and Usage Frequently Asked Questions](https://lux.collections.yale.edu/content/rights-info#faq-header)_ 

**Needed**

- [x] Core Team review
- [x] Define the feature requirements + capture questions - Rights WG
- Change "Subject To" label: to "Rights Status" (on works)
- Use Rights Status as Advanced Search Label
- Adv. Search Help should clarify: YUAG, YCBA only; YUL not making this assertion; YPM not passing works
- [x] wireframe/mockup with Help Text
- [x] Share wire/design with rights
- [x] Help Text - Sarah, Melissa, Rights WG
- [x] Change the scope of this ticket to only be the label change, the rest will be handled by ML ticket.  Track it here on this ticket: ML updates, label change, help text updates
- [ ] NEW mockup to show boolean option in Advanced Search and Help Text - Mike
(new public domain example: https://lux.collections.yale.edu/view/visual/edd6cdf3-bef1-4f61-92e2-ba8d06af9911)
- [ ] Confirm implementation plan (Dev Team)
- Change to Marklogic Configuration (for Adv. Search) - Peter (adding ticket) - Help Text and drop down option
- Front-end change: label is hard coded 
- Help Text updates (ML)
- Help Text updates (FAQ)
- [ ] New ML ticket - Peter

<img width="1103" alt="image" src="https://git.yale.edu/storage/user/2590/files/168ce0c2-cd8d-4cba-a969-5c50d3eddee1">

<img width="1136" alt="image" src="https://git.yale.edu/storage/user/2590/files/1d3c060e-e731-496a-b0a7-5d05451984db">

UAT: 
- Works Page (In copyright): https://lux.collections.yale.edu/view/visual/6aa8cbae-72ce-41c4-816e-d7af2602d428
- Works Page (public domain): https://lux.collections.yale.edu/view/visual/edd6cdf3-bef1-4f61-92e2-ba8d06af9911
- Advanced Search: https://lux.collections.yale.edu/view/results/works?q= should offer a dropdown for Rights Status.

**Comment by roamye** [comment]
Notes from UAT in old tix: 

@prowns:
"Notes from UAT

make Adv. search have sentence structure
need to align cc licenses with labels in user interface (not data)
users will need to be presented with a drop-down of options as there is no data
Questions

will it contain wikidata works? eg Daubigny's Garden
If this use case is making public domain available - can we turn this into a Boolean search? Would need to back to rights WG to answer

Note: this is not trivial"

**Comment by roamye** [comment]
@miketullo95 need to update the wireframe. it should be like "Has Digital Image" in advanced search. Boolean of Yes/No. Afterwards it can be brought to team meeting for discussion.

**Comment by miketullo95** [comment]
wireframe updated @roamye @prowns

**Comment by prowns** [comment]
Old mockups
![image](https://github.com/project-lux/lux-frontend/assets/160149818/6fdca359-4c59-43d0-9550-39ee75c24d9b)

![image](https://github.com/project-lux/lux-frontend/assets/160149818/73c877b1-7908-402f-9e5b-31b3516ada99)

**Comment by prowns** [comment]
> wireframe updated @roamye @prowns

Can you add the help text to mockup?  And put in the rest of the page context (search for works that...)?  Thanks!

**Comment by clarkepeterf** [comment]
@prowns the help text comes from MarkLogic. At current, links are not supported in the help text that comes from MarkLogic. The frontend could be modified to allow links to come from the MarkLogic configuration. Or we can change the help text such that it does not contain a link. Let me know what I should do. (This will be implemented in https://github.com/project-lux/lux-marklogic/issues/103)

**Comment by prowns** [comment]
@clarkepeterf - I just rewrote the help text to remove the link and refer to the View Advanced Search Help.  

Pardon the font, but something like this:
![image](https://github.com/project-lux/lux-frontend/assets/140099762/572f43b3-14b0-4a54-9d21-2fa6b85e3446)

@miketullo95 - any suggestions for improvement?

**Comment by miketullo95** [comment]
@prowns 
Only change i would have would be to link where it says " consult the Rights and Usage Frequently Asked Questions (Advanced Search Help) linked below.". I would just removed "linked below" and link the actual name of the content "Rights and Usage Frequently Asked Questions (Advanced Search Help)."

**Comment by roamye** [comment]
This issue is blocked by: https://github.com/project-lux/lux-marklogic/issues/103 which is still in backlog. Is this gets deployed would this only  show the options but not generate the search since the index properties are not done? 

@clarkepeterf @kamerynB

**Comment by kamerynB** [comment]
@roamye I'm also most certain this is being deployed today. @clarkepeterf Can you confirm?

**Comment by roamye** [comment]
@kamerynB yes that ticket is ready for this milestone (per teams discussion)

**Comment by roamye** [comment]
Approved per UAT
